### PR TITLE
chore(iec): move iec eips and bandwidths to services/iec

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:
move the file(iec/bandwidths and iec/eips) , fix error and  lint  
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx
fix some non-conformance functions and parameter names
**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
root@ecs-ruwenqiang:/home/huawei/go/src/github.com/huaweicloud/terraform-provider-huaweicloud# make testacc TEST="./huaweicloud/services/acceptance/iec" TESTARGS="-run TestAccBandWidthsDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iec -v -run TestAccBandWidthsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccBandWidthsDataSource_basic
=== PAUSE TestAccBandWidthsDataSource_basic
=== CONT  TestAccBandWidthsDataSource_basic
--- PASS: TestAccBandWidthsDataSource_basic (82.42s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iec       82.459s

```
```
root@ecs-ruwenqiang:/home/huawei/go/src/github.com/huaweicloud/terraform-provider-huaweicloud# make testacc TEST="./huaweicloud/services/acceptance/iec" TESTARGS="-run TestAccPublicIPsDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iec -v -run TestAccPublicIPsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccPublicIPsDataSource_basic
=== PAUSE TestAccPublicIPsDataSource_basic
=== CONT  TestAccPublicIPsDataSource_basic
--- PASS: TestAccPublicIPsDataSource_basic (72.23s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iec       72.269s
```

### CODE REVIEW

File: huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go [hunk 1]
```diff
@@ -105,22 +105,28 @@ func dataSourceIECBandWidthsRead(_ context.Context, d *schema.ResourceData, meta
...
+
 	if total > 0 {
 		d.Set("site_info", allBWs.BandWidth[0].SiteInfo)
 	}
 
 	return nil
 }
 
-func getLineName(operator iec_common.Operator) string {
+func getLineName(operator ieccommon.Operator) string {
```
Comment (Lance52259) [inline huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go:120]:
```suggestion
func getLineName(operator iecCommon.Operator) string {
```

File: huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 	"context"
+	"log"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
 	"github.com/chnsz/golangsdk/openstack/iec/v1/bandwidths"
-	iec_common "github.com/chnsz/golangsdk/openstack/iec/v1/common"
+	ieccommon "github.com/chnsz/golangsdk/openstack/iec/v1/common"
```
Comment (Lance52259) [inline huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go:12]:
```suggestion
	iecCommon "github.com/chnsz/golangsdk/openstack/iec/v1/common"
```

File: huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_eips_test.go [hunk 3]
```diff
@@ -7,56 +7,60 @@ import (
...
 	return func(s *terraform.State) error {
 		rs, ok := s.RootModule().Resources[n]
 		if !ok {
-			return fmtp.Errorf("Can't find IEC public IPs data source: %s", n)
+			return fmt.Errorf("can't find IEC public IPs data source: %s", n)
 		}
 
 		if rs.Primary.ID == "" {
-			return fmtp.Errorf("IEC public IPs data source ID not set")
+			return fmt.Errorf("IEC public IPs data source ID not set")
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_eips_test.go]:
```suggestion
			return fmt.Errorf("the ID of the IEC public IPs data source not set")
```

File: huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_eips_test.go [hunk 4]
```diff
@@ -68,12 +72,12 @@ resource "huaweicloud_iec_eip" "eip_test2" {
...
 
-func testAccIECEipsDataSource_basic() string {
+func testAccEipsDataSource_basic() string {
 	return fmt.Sprintf(`
 %s
 
-data "huaweicloud_iec_eips" "site" {
+data "huaweicloud_iec_eips" "test" {
   site_id = data.huaweicloud_iec_sites.sites_test.sites[0].id
 }
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/iec/data_source_huaweicloud_iec_eips_test.go]:
```suggestion
data "huaweicloud_iec_eips" "test" {
  depends_on = [
    huaweicloud_iec_eip.eip_test1,
    huaweicloud_iec_eip_eip_test2,
  ]
 
  site_id = data.huaweicloud_iec_sites.sites_test.sites[0].id
}
```

File: huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go [hunk 5]
```diff
@@ -105,22 +105,28 @@ func dataSourceIECBandWidthsRead(_ context.Context, d *schema.ResourceData, meta
...
-	if err := d.Set("bandwidths", iecBWs); err != nil {
-		return fmtp.DiagErrorf("Error saving IEC bandwidths: %s", err)
+	mErr := multierror.Append(d.Set("bandwidths", iecBWs))
+	if err := mErr.ErrorOrNil(); err != nil {
+		return diag.Errorf("error saving IEC bandwidths: %s", err)
 	}
+
 	if total > 0 {
 		d.Set("site_info", allBWs.BandWidth[0].SiteInfo)
 	}
```
Comment (Lance52259) [inline huaweicloud/services/iec/data_source_huaweicloud_iec_bandwidths.go]:
```suggestion
	mErr := multierror.Append(nil,
		d.Set("bandwidths", iecBWs)),
	}
	if total > 0 {
		mErr = multierror.Append(mErr, d.Set("site_info", allBWs.BandWidth[0].SiteInfo))
	}
	if err := mErr.ErrorOrNil(); err != nil {
		return diag.Errorf("error saving IEC bandwidths: %s", err)
	}
```
the error of the site information saving operation is not check.

**Comment by ruwenqiang123** [comment]
```
root@ecs-ruwenqiang:/home/huawei/go/src/github.com/huaweicloud/terraform-provider-huaweicloud# make testacc TEST="./huaweicloud/services/acceptance/iec" TESTARGS="-run TestAccBandWidthsDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iec -v -run TestAccBandWidthsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccBandWidthsDataSource_basic
=== PAUSE TestAccBandWidthsDataSource_basic
=== CONT  TestAccBandWidthsDataSource_basic
--- PASS: TestAccBandWidthsDataSource_basic (101.83s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iec       101.875s
```
```
root@ecs-ruwenqiang:/home/huawei/go/src/github.com/huaweicloud/terraform-provider-huaweicloud# make testacc TEST="./huaweicloud/services/acceptance/iec" TESTARGS="-run TestAccPublicIPsDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iec -v -run TestAccPublicIPsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccPublicIPsDataSource_basic
=== PAUSE TestAccPublicIPsDataSource_basic
=== CONT  TestAccPublicIPsDataSource_basic
--- PASS: TestAccPublicIPsDataSource_basic (78.25s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iec       78.298s
```

**Pull request merged**